xmtp_db/migrations/2025-08-20-182831_d14n_originator_id_group_messages/down.sql

```diff
@@ -0,0 +1 @@
+UPDATE group_messages SET originator_id = null WHERE originator_id IS 10;
```

Comment:
ideally i wanted to make the originator_id field on group_messages non-null, but that would require re-creating the VIEW. maybe will do it in another PR afterwards. having as little items as non-null simplifies code